api/v1beta1/watcher_webhook.go

```diff
@@ -36,6 +51,11 @@ func (r *Watcher) Default() {
 
```

Comment:
Don't we need to call the Default() func on the WatcherSpec here, inside the Default fucntion for Watcher?

r.Spec.Default()

Reply:
Done!